src/io/github/humbleui/canvas.clj

```diff
@@ -1,8 +1,14 @@
 (ns io.github.humbleui.canvas
   (:import
-    [io.github.humbleui.types IRect Rect RRect]
+    [io.github.humbleui.types Point IRect Rect RRect]
     [io.github.humbleui.skija BlendMode Canvas Font Paint]))
 
+(defn draw-polygon
+  ([^Canvas canvas floats-or-points ^Paint paint]
```

Comment:
Little uncertain if I should actually accept seqs in general instead of just arrays?

Reply:
I think so — they are way more natural for Clojure

Reply:
I've added support for seqs, they get converted to arrays now and passed through.